pkg/rest/utils.go

```diff
@@ -6,14 +6,82 @@ package rest
 
 import (
 	"context"
+	"encoding/json"
 	"net/http"
 
+	"github.com/labstack/echo/v4"
+
 	"github.com/IOTechSystems/go-mod-edge-utils/pkg/bootstrap/handlers"
 	"github.com/IOTechSystems/go-mod-edge-utils/pkg/common"
+	"github.com/IOTechSystems/go-mod-edge-utils/pkg/errors"
+	"github.com/IOTechSystems/go-mod-edge-utils/pkg/log"
 )
 
+// Versionable shows the API version in DTOs
+type Versionable struct {
+	ApiVersion string `json:"apiVersion" yaml:"apiVersion" toml:"apiVersion" validate:"required"`
```

Comment:
```suggestion
	ApiVersion string `json:"apiVersion" validate:"required"`
```
we don't need toml and yaml for now